KubeArmor/enforcer/appArmorProfile.go

```diff
@@ -675,21 +675,20 @@ func (ae *AppArmorEnforcer) GenerateProfileHead(numProcessWhiteList, numFileWhit
 	profileHead := "  #include <abstractions/base>\n"
 	profileHead = profileHead + "  umount,\n"
 
-	if defaultPosture.FileAction == "block" && !(numProcessWhiteList > 0 || numFileWhiteList > 0 || !fromSourceFile) {
-		profileHead = profileHead + "  file,\n"
-	} else if numProcessWhiteList == 0 && numFileWhiteList == 0 {
+	// Allow Access to Resource when
+	// -> Default Posture is not Block
+	// OR
+	// -> No allow policy AND No from source allow policy
+
+	if !(defaultPosture.FileAction == "block") || !(numProcessWhiteList > 0 || numFileWhiteList > 0 || !fromSourceFile) {
```

Comment:
it doesn't make sense since it will make all logs to alerts regardless of given actions if defaultPosture is not 'block'. In other words, even though we have an allow policy, the policy would be meaningless if defaultPosture is not "block".